example_notebooks/transformers/utils.py

```diff
@@ -3,31 +3,55 @@
 
 
 class LLMRunner:
-    def __init__(self, model_name='google/gemma-1.1-2b-it'):
+    def __init__(self, model_name="meta-llama/Llama-3.2-1B-Instruct"):
         self.tokenizer = AutoTokenizer.from_pretrained(model_name)
 
         self.model = AutoModelForCausalLM.from_pretrained(
             model_name,
             torch_dtype=torch.float16,
             device_map="auto",
-            trust_remote_code=True
         )
 
     def generate_response(self, prompts, logits_processor_list=None, max_tokens=1000):
         if logits_processor_list is None:
             logits_processor_list = []
 
-        input_ids = self.tokenizer(prompts, return_tensors='pt', padding=True)["input_ids"]
+        for prompt in prompts:
+            conversation = [
+                {"role": "system", "content": "You are a helpful assistant."},
+                {"role": "user", "content": prompt},
+            ]
+            inputs = self.tokenizer.apply_chat_template(
```

Comment:
Since chat template is applied now, can you please update the effected notebooks?